magma_access_gateway_configurator/__init__.py

```diff
@@ -22,20 +29,39 @@
 logger.addHandler(stdout_handler)
 
 
+PUBLIC_KEY_FILE = "/var/opt/magma/certs/gw_challenge.key"
+
+
 def main():
     args = cli_arguments_parser(sys.argv[1:])
     validate_args(args)
 
     aws_configurator = AGWConfigurator(args.domain, args.root_ca_path)
 
     logger.info("Starting Magma AGW configuration...")
+    if aws_configurator.control_proxy_configured:
+        print("Control Proxy configuration file already exists!")
```

Comment:
I used print instead of the logger, because the logger also logs to the journal. I figured this is not relevant for the journal. Now, when I think about it again, I'm actually having second thoughts - for potential debugging purposes it's good to leave a sign of system reconfiguration being done.
Changed.